"Validate node setup page mentions Docker as a container runtime" (#43011)
kubernetes/website: content/en/docs/setup/best-practices/node-conformance.md

standard Kubernetes node. At a minimum, the node should have the following
daemons installed:

> * Container Runtime (CRI-compatible container runtime example Docker, Containerd,CRI-O etc.)

Reviewer: ```suggestion
* CRI-compatible container runtimes such as Docker, containerd, and CRI-O
```
To be more concise

Author: > To be more concise

I also think this would be good and concise So commiting this changes.
